src/index.js

```diff
@@ -12,8 +12,9 @@ function parseHTML(document: Document, html: string): DocumentFragment {
 
 function serialize(form: HTMLFormElement): string {
   const params = new URLSearchParams()
-  for (const [name, value] of formDataEntries(form)) {
-    params.append(name, value)
+  const entries = 'entries' in FormData.prototype ? new FormData(form).entries() : formDataEntries(form)
+  for (const [name, value] of [...entries]) {
+    params.append(name, value.toString())
```

Comment:
Isn't value always a string?

```js
const form = new FormData()
form.append('a', 42)
[...form.entries()]
// => [["a", "42"]]
```

Reply:
https://xhr.spec.whatwg.org/#interface-formdata

- `iterable<USVString, FormDataEntryValue>;`
- `typedef (File or USVString) FormDataEntryValue;`

![image](https://user-images.githubusercontent.com/1153134/76669387-c3d57200-6561-11ea-8c0a-e51327640d83.png)

Reply:
This is an error in TypeScript's typing, where it considers `USVString` to be not-equivalent to `string`. Any DOM operation which accepts `USVString` will coerce the value `toString()`, but TypeScript does not allow for that behaviour. This is why we have `fetch(url.toString())` everywhere :(

Reply:
This project is using Flow and is giving the error that File isn’t String, which matches the spec I linked to.